enclave/src/c_api/c_api.cc

```diff
@@ -381,7 +381,8 @@ int add_client_key_with_certificate(char * cert,
         uint8_t* signature,
         size_t sig_len) {
     API_BEGIN();
-    EnclaveContext::getInstance().decrypt_and_save_client_key_with_certificate(cert, cert_len,data, data_len, signature, sig_len);
+    EnclaveContext::getInstance().decrypt_and_save_client_key_with_certificate(cert, cert_len, data, data_len, signature, sig_len);
+    EnclaveContext::getInstance().share_symm_key_and_nonce();
```

Comment:
I don't think it's the right place to call `share_symm_key_and_nonce`. In fact, this ecall (`add_client_key_with_certificate`) is probably not the right place to put this function call either, but I wanted to avoid adding another ecall just to broadcast the symmetric key and nonce. Any ideas on how we can do this? Maybe rename the ecall?

Reply:
Hmm, I suppose `share_symm_key_and_nonce` only needs to be done once, right? So yeah, it shouldn't be invoked every time a client key is added.
I think a better place for this is the rabit init ecall -- all the enclaves attest each other + set up TLS channels, and also do `share_symm_key_and_nonce`.

Reply:
Moved to rabit init